examples/snippets/data_io/df_connect/streamable_check.py

```diff
@@ -0,0 +1,31 @@
+"""Snippet to add an explicit check to validate that a dataset is streamable using df_connect
+Do this by passing in a Dataset object rather than just the dataset name"""
```

Comment:
I don't think this second line is necessary, also slight wording change:
```suggestion
"""Snippet to add an explicit check to validate that a dataset is streamable when exporting with df_connect"""
```